src/api.js

```diff
@@ -545,14 +543,14 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
             pos = this.pos;
             this.pos += 1;
         }
-        var bytes = intArrayFromString(string);
-        var strptr = allocate(bytes, ALLOC_NORMAL);
+        var length = lengthBytesUTF8(string);
+        var strptr = stringToNewUTF8(string);
```

Comment:
isn't this going to go over the string twice instead of just once ?

Reply:
Turns out we can avoid `stringToNewUTF8` by passing -1 instead: https://www.sqlite.org/c3ref/bind_blob.html

If a non-negative fourth parameter is provided to sqlite3_bind_text() or sqlite3_bind_text16() or sqlite3_bind_text64() then that parameter must be the byte offset where the NUL terminator would occur assuming the string were NUL terminated.